src/app/__tests__/App.test.tsx

```diff
@@ -37,7 +41,7 @@ describe('src/app/App', () => {
         });
         (fetchRBAC as jest.Mock).mockImplementation(() => promise);
         render(
-            <App/>,
+            <IntlProvider locale={ navigator.language } messages={ messages }><App /></IntlProvider>,
```

Comment:
This is using `AppWrapper` already, do we need the `IntlProvider` here (and the same on the rest of the file)

Reply:
yeah we do sadly, wish the wrapper was enough but it seems we also need em on each individual render

Reply:
:cry: